internal/service/api/requests/get_role.go

```diff
@@ -0,0 +1,19 @@
+package requests
+
+import (
+	"net/http"
+
+	"gitlab.com/distributed_lab/urlval"
+)
+
+type GetRoleRequest struct {
+	AccessLevel *string `filter:"accessLevel"`
+}
+
+func NewGetRoleRequest(r *http.Request) (GetRoleRequest, error) {
+	var request GetRoleRequest
+
+	err := urlval.Decode(r.URL.Query(), &request)
+
+	return request, err
```

Comment:
1. Wrap error 
2. Use `validation.Errors` for bad request errors in order to properly parse them in FE part